components/File/README.md

```diff
@@ -21,7 +21,7 @@ The file download consists of:
 
 ## (Interactive) states
 
-The file download contains the states normal, hover and focus.
+The file contains the states normal, hover and focus.
 
 ## Design properties
 
```

Comment:
U can use the readme from here: https://www.figma.com/file/JpoY3waVoQGlLQzQXTL9nn/Den-Haag-Design-System?type=design&node-id=5418-16936&mode=dev